src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -31,6 +31,7 @@
 # include <Inventor/SoPickedPoint.h>
 # include <Inventor/details/SoPointDetail.h>
 # include <Inventor/events/SoKeyboardEvent.h>
+# include <Inventor/nodes/SoBaseColor.h>

```

Comment:
Is this needed ?

Reply:
SoSeparator* createGridPart(double Step, int numberSubdiv, bool divLines, bool subDivLines, int pattern, SoBaseColor* color, int lineWidth = 1);

Reply:
??? Isn't that moved to the CoinManager already ?

Reply:
Oh missed the file name